src/manage_sql_nvts.c

```diff
@@ -1415,6 +1415,7 @@ manage_update_nvt_cache_osp (const gchar *update_socket)
 {
   osp_connection_t *connection;
   gchar *db_feed_version, *scanner_feed_version;
+  GSList *scanner_prefs;
```

Comment:
This is only used in one block so it should be declared in that block.

Reply:
I've moved the variable in daa969520825068f563c0a03752b730ba7e0c4ab.